### PR TITLE
fix(contributors): new contributors

### LINKED ISSUE #852: Add team to the contributors list

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #852 

### What does this PR do?
The project uses the all-contributors, however the files where missing.
https://github.com/all-contributors/all-contributors

### How should this be tested?
- Make run

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,407 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (leisterfrancisco) [inline yarn.lock:1]: Is this file necessary?

**Review by leisterfrancisco — APPROVED** [review]

**Review by steph222 — APPROVED** [review]
